### PR TITLE
added garbarge

### PR DESCRIPTION
**Pull request by IndianHills-CSD**
💩

### CODE REVIEW

**Comment by Nceastep89** [comment]
looks like 💩

File: README.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ Nathan Eastep
 Derek Oppenheimer
 Alex Dawson
 
+asldfjas;lfkjda;lksfj;lk
```
Comment (IndianHills-CSD) [inline README.md:7]: did you have a stroke?

Comment (adawson21) [inline README.md:7]: bad code

Comment (hotheadheather) [reply README.md:7]: no

**Review by IndianHills-CSD — COMMENTED** [review]
@hotheadheather could take a look at this pull request?

**Comment by hotheadheather** [comment]
not ready

File: README.md (hunk 1 above)
Comment (adawson21) [reply README.md:7]: yes

**Comment by hotheadheather** [comment]
i changed my mind

File: README.md [hunk 2]
```diff
@@ -4,3 +4,4 @@ Nathan Eastep
 Derek Oppenheimer
 Alex Dawson
 
+Kelsey Van Antwerp
```
Comment (adawson21) [inline README.md:7]: Seems kinda sus.
I don't remember you being a part of our team

**Pull request closed without merging**